src/webhooks/registry.ts

```diff
@@ -144,8 +152,10 @@ const WebhooksRegistry: RegistryInterface = {
     webhookHandler,
   }: RegisterOptions): Promise<RegisterReturn> {
     const client = new GraphqlClient(shop, accessToken);
-    const address = `https://${Context.HOST_NAME}${path}`;
-
+    const address =
+      deliveryMethod && deliveryMethod === DeliveryMethod.EventBridge
```

Comment:
We don't really need to check if `deliveryMethod` is set here since it defaults to `Http`.

Reply:
Good point, removed.